src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -70,7 +121,7 @@ const CustomFilterModal = ({
 
         <hr className="my-1" />
         <p className="fs-6 mb-0 ms-4 cursor-pointer text-filters-container">
```

Comment:
cursor-pointer class is not exist , why use it

Reply:
resolved